azure/citus-bot.sh

```diff
@@ -7,6 +7,22 @@ set -e
 # echo commands
 set -x
 
+ssh_execute() {
```

Comment:
might be useful to have a comment on the function (probably single line), explaining the reason for a function